### PR TITLE
PLATUI-2109 retry flickering tests; fix some given/when/then oddities

### PR DESCRIPTION
**Pull request by ellamdav**
From a bit of searching, it seems that the 502 Bad Gateway error may indicate a cache problem with the browser.  Rather than go down a rabbit hole trying to diagnose and fix that, I've added code to retry any tests that fail and have the `Retryable` tag.

### CODE REVIEW

File: test/acceptance/specs/CookieSettingsPageSpec.scala [hunk 1]
```diff
@@ -115,11 +116,6 @@ class CookieSettingsPageSpec extends BaseAcceptanceSpec {
 
     And("the dataLayer contains the 'trackingConsentSettingsAccepted' event")
     settingsAllowedGtmEvent should not be null
-
```
Comment (ellamdav) [inline test/acceptance/specs/CookieSettingsPageSpec.scala:118]: this is outside the scope of this test of GTM events - there's a specific test for setting the cookie (the next one)

**Review by kyle-bowden — APPROVED** [review]
